### PR TITLE
Style select target funds tiles

### PR DESCRIPTION
**Pull request by nickymiron**
select-funds

### CODE REVIEW

File: src/steps/selectSources/targetFundSelector/TargetFundSelector.scss [hunk 1]
```diff
@@ -12,6 +38,15 @@
     border-color: transparent;
   }
 
+  &__title {
+    font-size: 16px;
+    font-weight: 600;
+  }
```
Comment (Tankenstein) [inline src/steps/selectSources/targetFundSelector/TargetFundSelector.scss]: Don't define these yourself pls, use the predefined styles.

File: src/steps/selectSources/targetFundSelector/TargetFundSelector.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import React, { PropTypes as Types } from 'react';
 import { Message } from 'retranslate';
 
+import { InfoTooltip } from '../../../common';
+import TargetFundTooltipBody from '../../transferFutureCapital/targetFundTooltipBody';
+
 import './TargetFundSelector.scss';
+import checkImage from '../../success/success.svg';
```
Comment (Tankenstein) [inline src/steps/selectSources/targetFundSelector/TargetFundSelector.js]: Maybe worth moving this to common or something?

Comment (nickymiron) [inline src/steps/selectSources/targetFundSelector/TargetFundSelector.js]: Sure, lets do it tomorrow. Want to merge and go home now.

File: src/steps/selectSources/targetFundSelector/TargetFundSelector.scss [hunk 3]
```diff
@@ -4,6 +4,32 @@
   border: solid 1px #cccccc;
   cursor: pointer;
   transition: border-color .15s ease-out;
+  position: relative;
+  overflow: hidden;
+
+  &__corner-check {
```
Comment (Tankenstein) [inline src/steps/selectSources/targetFundSelector/TargetFundSelector.scss]: I would rather do this with pseudo elements.

**Pull request merged**